sdk/cosmos/src/requests/create_slug_attachment_builder.rs

```diff
@@ -13,7 +14,7 @@ where
     ContentTypeSet: ToAssign,
 {
     attachment_client: &'a AttachmentClient,
-    body: Option<&'b [u8]>,
+    body: Option<Bytes>,
```

Comment:
I'm not sure about this change. I don't really know how CreateSlugAttachmentBuilder is used. If It is reused, this should be a `&Bytes` then cloned.

Reply:
According to the docs ([https://docs.rs/bytes/1.0.1/bytes/](https://docs.rs/bytes/1.0.1/bytes/)) `bytes` uses ref counting so cloning should be relatively cheap. I'd go with the owned struct.